dpdispatcher/machine.py

```diff
@@ -15,6 +15,7 @@
 """
 
 script_env_template="""
+export FLAG_TASK_FAIL=0
```

Comment:
I think we don't need `export` here, as we only use it in this script.
```suggestion
FLAG_TASK_FAIL=0
```